data-sets-api-server/src/main/java/org/zowe/data/sets/services/zosmf/ZosmfDataSetService1.java

```diff
@@ -0,0 +1,27 @@
+/*

```

Comment:
For consistency this file should be name ZosmfDataSetService**V**1.java

and in line the v2 one should be renamed to include **V**

Reply:
done